src/pudl/etl/__init__.py

```diff
@@ -45,8 +45,16 @@
     *load_assets_from_modules([pudl.transform.ferc714], group_name="clean_ferc714"),
     *load_assets_from_modules([glue_assets], group_name="glue"),
     *load_assets_from_modules([static_assets], group_name="static"),
-    *load_assets_from_modules([pudl.output.denorm_eia], group_name="denorm_eia"),
-    *load_assets_from_modules([pudl.output.denorm_ferc1], group_name="denorm_ferc1"),
+    *load_assets_from_modules(
+        [
+            pudl.output.eia,
+            pudl.output.eia860,
+            pudl.output.eia923,
+            pudl.output.eia_bulk_elec,
+        ],
+        group_name="denorm_eia",
+    ),
+    *load_assets_from_modules([pudl.output.new_ferc1], group_name="new_ferc1"),
```

Comment:
I kind of doubt we're going to stick with the `denorm_` naming for the tables themselves, since I think those are the tables we want most people to use (so they should probably have the clearest, simplest, unmodified names) but if you want to switch the asset group name go for it!